cmake/target/dict.cmake

```diff
@@ -17,19 +17,33 @@ function(dictgen MODULE_NAME AI_XML MOD_DEPS)
   string(REGEX REPLACE "Ai.xml" "Dictionary.xml" DICT_XML "${AI_XML}")
   string(REGEX REPLACE "Ai.xml" "Ac" AC_BASE "${AI_XML}")
   string(REPLACE ";" ":" FPRIME_BUILD_LOCATIONS_SEP "${FPRIME_BUILD_LOCATIONS}")
-  # Building here with a fake dictionary output file, so this command will always rebuild.
+  get_filename_component(DICT_XML_NAME ${DICT_XML} NAME)
```

Comment:
Does this imply that you are copying the dictionary straight to the package_gen?

Reply:
Yes, it's created, copied into build artifacts, then deleted from the source tree.